src/Playwright.Tests/PageRequestContinueTests.cs

```diff
@@ -90,5 +90,41 @@ public async Task ShouldAmendPostData()
             );
             Assert.AreEqual("doggo", requestTask.Result);
         }
+
+        public async Task ShouldNotThrowWhenContinuingWhilePageIsClosing()
```

Comment:
Does this need at least a `[PlaywrightTest]` ?
```suggestion
        [PlaywrightTest]
        public async Task ShouldNotThrowWhenContinuingWhilePageIsClosing()
```

Reply:
good catch, fixed.